core/src/main/java/org/infinispan/interceptors/locking/AbstractLockingInterceptor.java

```diff
@@ -35,16 +38,19 @@
  * Base class for various locking interceptors in this package.
  *
  * @author Mircea Markus
- * @deprecated Since 8.2, no longer public API.
  */
-@Deprecated
-public abstract class AbstractLockingInterceptor extends CommandInterceptor {
-   private boolean trace = getLog().isTraceEnabled();
+public abstract class AbstractLockingInterceptor extends DDSequentialInterceptor {
+   private static final Log log = LogFactory.getLog(AbstractLockingInterceptor.class);
```

Comment:
The rest of the code was still using `getLog()` everywhere, so I just removed the field and the method body.